Reply/app/src/main/java/com/materialstudies/reply/ui/compose/ComposeFragment.kt

```diff
@@ -110,4 +141,88 @@ class ComposeFragment : Fragment() {
             addView(chipBinding.root)
         }
     }
+
+    /**
+     * Add a card for the given [Account] to the recipients card.
+     */
+    private fun addRecipientsCard(acnt: Account) {
+        binding.recipientCardView.run {
+            val cardBinding = ComposeRecipientCardBinding.inflate(
+              LayoutInflater.from(context),
+              this,
+              false
+            ).apply {
+                account = acnt
+            }
+
+            addView(cardBinding.root)
+        }
+    }
+
+    private fun prepareTransitions() {
+        postponeEnterTransition()
+    }
+
+    private fun startTransitions() {
+        binding.executePendingBindings()
+        // Delay creating the enterTransition until after we have inflated this Fragment's binding
+        // and are able to access the view to be transitioned to.
+        enterTransition = MaterialContainerTransform().apply {
+            // Manually add the Views to be shared since this is not a standard Fragment to Fragment
+            // shared element transition.
+            startView = requireActivity().findViewById(R.id.fab)
+            endView = binding.emailCardView
+            duration = resources.getInteger(R.integer.reply_motion_duration_large).toLong()
+            interpolator = requireContext().themeInterpolator(R.attr.motionInterpolatorPersistent)
+        }
+        returnTransition = Slide().apply {
+            duration = resources.getInteger(R.integer.reply_motion_duration_medium).toLong()
+            interpolator = requireContext().themeInterpolator(R.attr.motionInterpolatorOutgoing)
+        }
+        startPostponedEnterTransition()
+    }
```

Comment:
remove `prepareTransitions` and `startTransitions`? looks like they were copied from `onViewCreated` above.